docs/ruleset_functions/is_zone_mechanically_heated_and_not_cooled.md

```diff
@@ -0,0 +1,43 @@
+# is_zone_mechanically_heated_and_not_cooled
+**Schema Version:** 0.0.23  
+
+**Description:** determines whether a zone is mechanically heated, but not cooled.  Checks for transfer air
+
+**Inputs:**
+- **RMI**
+- **zone**
+
+**Returns:**  
+- **result**: a boolean - TRUE or FALSE
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- set a boolean has_heating_system to false: `has_heating_system = FALSE`
+- set a boolean has_cooling_system to false: `has_cooling_system = FALSE`
+- set result to FALSE: `result = FALSE`
+- get the hvac systems serving the zone by using the get_list_hvac_systems_associated_with_zone function: `list_hvac_systems = get_list_hvac_systems_associated_with_zone(RMI,zone)`
+	- loop through each HeatingVentilatingAirconditionsSystem system: `for system in list_hvac_systems:`
```

Comment:
it would help to clarify that line 21 is looping through each zone and for each zone, it is looking through each HVAC system associated with the zone. I incorrectly read this to mean that we're looking through each HVAC system in the RMI.